src/main/java/blackjack/domain/card/CardDeck.java

```diff
@@ -0,0 +1,73 @@
+package blackjack.domain.card;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CardDeck {
+    private static final int DECK_SIZE = 52;
```

Comment:
```suggestion
    private static final int DECK_SIZE = Suit.values().length * Rank.values().length;
```
물론 트럼프 카드가 문양 4개, 숫자 13개로 고정되어서 덱 사이즈가 52에서 변경될 일이 거의 없기는 해요. 
다만 이 미션 자체가 '커스텀 된 블랙잭'인 만큼, 카드 문양과 숫자를 제한하는 요구사항 역시 충분히 발생할 수 있을 것 같더라고요. 이럴 경우 리터럴값으로 값이 고정되는 것보다 위같은 방식을 통해 더 유연하게 대응하면 어떨까 싶어 제안 드려봅니다 :)